java/sync-map-and-scene-viewpoints/README.md

```diff
@@ -1,20 +1,35 @@
-# Sync Map and Scene Viewpoints
-This sample demonstrates how to keep the viewpoints of multiple map or scene views in sync, so that navigating one view immediately updates the others.
+# GeoView viewpoint synchronization
 
-![Sync Map and Scene Viewpoints App](sync-map-and-scene-viewpoints.png)
+Keep the view points of two views (e.g. MapView and SceneView) synchronized with each other.
+
+![Image of geo view viewpoint synchronization](sync-map-and-scene-viewpoints.png)
+
+## Use case
+
+You might need to synchronize GeoView viewpoints if you had two map views in one application - a main map and an inset. An inset map view could display all the layers at their full extent and contain a hollow rectangular graphic that represents the visible extent of the main map view. As you zoom or pan in the main map view, the extent graphic in the inset map would adjust accordingly.
 
 ## How to use the sample
-Pan, zoom, and rotate the map or scene view. The other view will update automatically to match your navigation. Note that maps are 2D while scenes are 3D, so the results may not look identical, but the centers and scales will be kept the same.
+
+Interact with the MapView or SceneView by zooming or panning. The other MapView or SceneView will automatically focus on the same viewpoint.
 
 ## How it works
-`GeoView`, the parent class of both `MapView` and `SceneView`, has a property `ViewpointChangeListener` which is called each time the viewpoint updates. Inside this listener we get the viewpoint of the `GeoView` by calling `.getCurrentViewpoint(Viewpoint.Type.CENTER_AND_SCALE)`. We then pass that viewpoint into `setViewpoint(...)` on the other `GeoView`, thus synchronizing both views.
+
+1. Wire up the `ViewpointChanged` event handler for both geo views.
+2. In each event handler, get the current viewpoint from the geo view that is being interacted with and then set the viewpoint of the other geo view to the same value.
 
 ## Relevant API
```

Comment:
we only include Class names in the Relevant API sections, so let's take out the method calls: it's not clear what they relate to. The user can see in the code what the relevant methods are, since the readmes are intended as an overview to the process.